CI/setup-windows-sdk.sh

```diff
@@ -235,16 +224,7 @@ fi
 # itself runs in a Lua 5.4 environment) - otherwise one has to do the same thing
 # for EVERY luarock!
 
-# Save the wanted Luarocks 5.1 command so it can be used repeatedly next
-# this uses the "system" (shared between all users) tree, using the --local
-# option does NOT work, but the alternative --tree "user" (the literal string
-# user NOT the user's name) does for "per user" luarocks as the previous use
-# of sed above has "fixed" things:
```

Comment:
I was going to replace this with "# Use the correct version of Lua when installing the luarocks" but then that was a redundant comment because it's exactly what the line does. In the end, the comment isn't adding all that much but it does take time and energy to read through it.